samples/storageclass/storageclass-xfs.yaml

```diff
@@ -28,8 +28,8 @@ parameters:
   # format options to pass to mkfs
   # Allowed values: A string dictating the fs options you want passed
   # Optional: true
-  # Delete the line below if you do not plan on using mkfsFormatOption
-  mkfsFormatOption: "<mkfs_format_option>" # Insert file system format option
+  # Uncomment the line below if you want to use mkfsFormatOption
```

Comment:
Uncomment and enter <mkfs_format_option> values to use for the line below if you want to use mkfsFormatOption
we dont want an empty
mkfsFormatOption:  < -like you had ?

Reply:
Ahh true I'll add that

Reply:
@nb950 actually, if you look at the line with `mkfsFormatOption`, there is already a comment there saying to add in a value. So I think this is OK as it is.